games/static/games/proto.js

```diff
@@ -7,22 +7,40 @@ class Game {
     this.restartButton = document.querySelector("#restart");
     this.isRunning = false
   }
+  win(board) {
+    return !board.containsAliens()
+  }
+  checkLose(board, gameId) {
+    if (this.aliensHitGround(board)) {
+      return this.lose(gameId)
+    }
+    
+  }
+  aliensHitGround(board) {
+    return board.squares.filter(square => square.isGround() && square.hasAlien())[0]
```

Comment:
you can use the `.some()` function:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some